src/components/home/sections/Volunteer.js

```diff
@@ -14,52 +9,24 @@ const SECTION_TITLE = 'An invitation from our board members'
 
 export const Volunteer = () => {
   return (
-    <section id='volunteer'>
-
-      <div className='volunteer-content'>
+    <div>
+      <section id='volunteer'>
 
-        <h1 className='volunteer-title'>{SECTION_TITLE}</h1>
-        <p>{CONTENT1}</p>
-        <a href='https://docs.google.com/forms/d/e/1FAIpQLScEOOuZ1iZw9IWbsiWc1AX3OHhFQ4wpxYs_TZHyykx2glsrFw/viewform' target='_blank' rel='noopener noreferrer'><button className='btn-container'>Sign Up</button></a>
-        <p>{CONTENT2}</p>
-        <p>{CONTENT3}</p>
+        <div className='volunteer-content'>
 
-        <a href='https://docs.google.com/forms/d/e/1FAIpQLSd9N1_wWETbeMs7UAxzUTCMl0FhCnVneozlA-EMAMQXSBFy3A/viewform' target='_blank' rel='noopener noreferrer'>
-          <button className='btn-container'>volunteer with us</button>
-        </a>
-      </div>
+          <h1 className='volunteer-title'>{SECTION_TITLE}</h1>
```

Comment:
Is this class name being used in css anywhere?

Reply:
I believe this is being used. If not it was intended as I was not on the team when the volunteer sections were created.

Reply:
No need to keep it if it's not being used.